### PR TITLE
Add support of Fastly directors

### PR DESCRIPTION
**Pull request by cono**

### CODE REVIEW

File: library/fastly_service.py [hunk 1]
```diff
@@ -346,7 +379,8 @@ class FastlyVclSnippet(FastlyObject):
     schema = {
         'name': dict(required=True, type='str', default=None),
         'dynamic': dict(required=False, type='int', default=0),
-        'type': dict(required=False, type='str', default='init'),
+        'type': dict(required=False, type='str', default='init',
+                     choices=['init', 'recv', 'hit', 'miss', 'pass', 'fetch', 'error', 'deliver', 'log', 'none', None]),
```
Comment (cono) [inline library/fastly_service.py]: Ah, sorry forgot to mention, one more change is validation of vclsnippet.type field

**Comment by fstehle** [comment]
Thanks a lot for your contribution.
Would you mind to add the director documentation to the [README](https://github.com/Jimdo/ansible-fastly/blob/master/README.md)?

File: library/fastly_service.py (hunk 1 above)
Comment (fstehle) [reply library/fastly_service.py]: Oh, I missed this, can you create a separate PR for this please as it has nothing to do with directors?

**Comment by cono** [comment]
@fstehle sorry not familiar with this "Change requests". Do I need to do anything on my side?

**Comment by fstehle** [comment]
@cono You can just create a new branch & pull request with the change.

**Pull request merged**